posix/clone

```diff
@@ -2,6 +2,14 @@
 
 set -e
 
+chmod +x copy-file-content
```

Comment:
don't do it here, do it in your local and git commit it

Reply:
done